docs/topics/db/sql.txt

```diff
@@ -334,6 +334,9 @@ Also note that Django expects the ``"%s"`` placeholder, *not* the ``"?"``
 placeholder, which is used by the SQLite Python bindings. This is for the sake
 of consistency and sanity.
 
+DatabaseWrapper can also pass any argument for the cursor() calls to the underlying database adapter. This makes it possible to use the adapter fully at the low level.
```

Comment:
DatabaseWrapper isn't something that users will be familiar with.  How about giving an example use case that demonstrates the behavior. 

Have you read https://docs.djangoproject.com/en/dev/internals/contributing/writing-documentation/? There are a lot of issues: 
- lines not wrapped at 79 characters
- no backticks around backticks like code things like `cursor()`
- no versionadded annotation.
- don't use 2 blank lines in a row